src/test/java/teammates/test/cases/webapi/EnrollStudentsActionTest.java

```diff
@@ -0,0 +1,208 @@
+package teammates.test.cases.webapi;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.http.HttpStatus;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.exception.InvalidHttpRequestBodyException;
+import teammates.common.util.Const;
+import teammates.ui.webapi.action.EnrollStudentsAction;
+import teammates.ui.webapi.action.JsonResult;
+import teammates.ui.webapi.output.StudentData;
+import teammates.ui.webapi.output.StudentsData;
+import teammates.ui.webapi.request.StudentsEnrollRequest;
+
+
+/**
+ * SUT: {@link EnrollStudentsAction}.
+ */
+public class EnrollStudentsActionTest extends BaseActionTest<EnrollStudentsAction> {
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.STUDENTS;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return PUT;
+    }
+
+    @Override
+    @Test
+    public void testExecute() throws Exception {
+        // See test cases below.
+    }
+
+    @Test
+    public void testExecute_withNewStudent_shouldBeAddedToDatabase() {
+        String courseId = typicalBundle.students.get("student1InCourse1").getCourse();
+        StudentAttributes newStudent = getTypicalNewStudent(courseId);
+        StudentsEnrollRequest req = prepareRequest(Arrays.asList(newStudent));
+
+        loginAsInstructor(typicalBundle.instructors.get("instructor1OfCourse1").getGoogleId());
+        List<StudentData> enrolledStudents = executeActionAndReturnResults(courseId, req);
+
+        assertEquals(1, enrolledStudents.size());
+        verifyStudentInDatabase(newStudent, enrolledStudents.get(0).getCourseId(), enrolledStudents.get(0).getEmail());
```

Comment:
You should verify at least have one test case verifying `enrolledStudents` has correct content?